.github/DISCUSSION_TEMPLATE/sandbox.yml

```
@@ -0,0 +1,52 @@
+    attributes:
+      label: Organization
+      description: "Enter the name of the organization you represent"
+      placeholder: "e.g., XYZ"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: product_name
+    attributes:
+      label: Product
+      options: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)